SayTheirNames/Source/Utils/Theme/Theme.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  Theme.swift
+//  SayTheirNames
+//
+//  Created by Berta Devant on 05/06/2020.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+/// A collection of common constants for margins and sizes
+enum Theme {
+    // MARK: - Components
+    enum Components {
+        // MARK: - Padding
+        enum Padding {
+            //For cohesive design Most padding used are multiple of 8
+            static let tiny: CGFloat = 4
+            static let small: CGFloat = 8
+            static let medium: CGFloat = 16
+            static let large: CGFloat = 32
+        }
+        // MARK: - Button
+        enum Button {
+            enum Size {
+                static let medium: CGSize = .init(width: 40, height: 40)
+            }
+        }
+    }
+    
+    // MARK: - Screens
+    enum Screens {
```

Comment:
Nicely done! 

Don't think we should separate styling per screen. We could drop `enum home` & `enum screens` this way we have standard sizes across screens. Should we need custom per screen we can do `static let homePeopleHeight = 170`

Reply:
I was thinking the same way but current screens really do use different sizes and I am not sure if I should be the one to standardize things, so i wanted to have all put together and then on a separate PR talk to design and get standards!

Reply:
Sounds like a perfect case for a follow-up to-do indeed - that would be a nice small ticket to work on